Supervision Work/src/uk/ac/cam/ap886/oopjava/supervisionProjects/Hangman/Dictionary.java

```diff
@@ -0,0 +1,7 @@
+package uk.ac.cam.ap886.oopjava.supervisionProjects.Hangman;
+
+public interface Dictionary {
```

Comment:
Oh yeah sorry that's me being dumb, of course this isn't a dictionary in the "usual" sense, sorry about that